GDJS/Runtime/runtimeobject.js

```diff
@@ -1097,7 +1111,7 @@ gdjs.RuntimeObject.prototype.getTimerElapsedTimeInSeconds = function(timerName)
 
 /**
  * Separate the object from others objects, using their hitboxes.
- * @param objects Objects
+ * @param objects Objects {@link Array<RuntimeObject>}
```

Comment:
Why a link? Should that bit a type instead?

Reply:
Sorry, I just made that as a little note to myself. This was really for the type definitions and not much for the JSDoc. I planned to do the JSDoc in a other pr